src/server/helpers/adminLogs.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2023 Shivam Awasthi
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {AdminActionType} from '../../common/helpers/privileges-utils';
+
+
+export type AdminLogDataT = {
+	actionType: AdminActionType,
+	admin?: any,
+	adminId: number,
+	id?: number,
+	newPrivs: number,
+	note: string,
+	oldPrivs: number,
+	targetUser?: any,
+	targetUserId: number,
+	time?: any
+};
```

Comment:
I agree, but would suggest to do this as a separate PR, so we can move forward.
Since it will require publishing a new version of the bookbrainz-data package, we might as well collect all these types that need to be moved, and do as many as possible in one go to reduce the back-and-forth work.